ext/pdo/php_pdo_driver.h

```diff
@@ -577,6 +577,8 @@ struct _pdo_stmt_t {
 	 * bindParam() for its prepared statements, if false, PDO should
 	 * emulate prepare and bind on its behalf */
 	unsigned supports_placeholders:2;
+	/* if true, the statement SQL should be parsed using MySQL syntax */
+	unsigned mysql_compat:3;
 
 	unsigned _reserved:29;
```

Comment:
These numbers (`3` and `29`) designate how many bits are to be used for these fields (this is called a bitfield). `mysql_compat` only needs a single bit, and `_reserved` needs to be decremented, so:
```suggestion
	unsigned mysql_compat:1;

	unsigned _reserved:28;
```